onboard-julia/login-page.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import {
+  View,
+  Text,
+  TextInput,
+  StyleSheet,
+  TouchableOpacity,
+} from 'react-native';
+
+export function Login(): React.JSX.Element {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.title}>Bem-vindo(a) à Taqtile!</Text>
+      <Text>E-mail</Text>
+      <TextInput style={styles.input} />
+      <Text>Senha</Text>
+      <TextInput style={styles.input} secureTextEntry />
+      <TouchableOpacity style={styles.button}>
+        <Text style={styles.buttonText}>Entrar</Text>
+      </TouchableOpacity>
+    </View>
```

Comment:
nos componentes é interessante a gente adicionar uma separação igual em  uma função js normal pra facilitar a leitura, da pra fazer uma divisão por bloco visuais. ex:
```suggestion
    <View style={styles.container}>
      <Text style={styles.title}>Bem-vindo(a) à Taqtile!</Text>
      
      <Text>E-mail</Text>
      <TextInput style={styles.input} />

      <Text>Senha</Text>
      <TextInput style={styles.input} secureTextEntry />
      
      <TouchableOpacity style={styles.button}>
        <Text style={styles.buttonText}>Entrar</Text>
      </TouchableOpacity>
    </View>
```

A ideia é melhorar a legibilidade do código e isso pode ser aplicado em vários contextos